server/test/endpoint_clients.test.js

```diff
@@ -1,86 +1,103 @@
 const { describe, it } = require('mocha')
-// const fetch = require('node-fetch')
+const fetch = require('node-fetch')
 const assert = require('assert')
-require('dotenv').config()
-// const { PGPool } = require('database/layer/pg_pool')
-// const pool = new PGPool()
+const { getAdmin1IdPToken } = require('./lib/tokens')
+const { getServerUrl } = require('./lib/urls')
+const dbAdminRole = require('database/layer/admin-client')
 
-// const { getAdmin1IdPToken, getAdmin2IdPToken } = require('./lib/tokens')
-// const { getServerUrl } = require('./lib/urls')
-
-describe('Admin Clients Endpoint Test', function () {
-  // const url = `${getServerUrl()}/clients`
-
-  it.skip('Should identify these tests are commented out during minimal spec development', function () {
-    assert(true, 'Tests not run')
+describe('Admin Clients Endpoint Test', () => {
+  let userAccessToken = ''
+  const url = getServerUrl()
+  let requestHeaders
+  before(async function () {
```

Comment:
If we are not using function context `this`, we can use arrow function instead. We can be consistent in the file. 
```suggestion
  before(async () => {
```